third_party/bigframes_vendored/ibis/backends/bigquery/registry.py

```diff
@@ -38,13 +38,25 @@ def _quantile(translator, op: ibis_reductions.Quantile):
     return f"PERCENTILE_CONT({arg}, {quantile})"
 
 
+def _array_aggregate(translator, op: vendored_ibis_ops.ArrayAggregate):
```

Comment:
Do we have an issue upstream for this? If so, please add a comment with a link so that we can remove this once its resolved. If not, please file one.